pkg/protoc/providers.go

```diff
@@ -0,0 +1,65 @@
+package protoc
+
+import (
+	"sync"
+
+	"github.com/google/go-cloud/wire"
+	"github.com/izumin5210/gex"
+	"github.com/izumin5210/gex/pkg/tool"
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+	"k8s.io/utils/exec"
+
+	"github.com/izumin5210/grapi/pkg/cli"
+)
+
+var (
+	gexCfg   *gex.Config
+	gexCfgMu sync.Mutex
+
+	toolRepo   tool.Repository
+	toolRepoMu sync.Mutex
+)
+
+func ProvideGexConfig(
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function ProvideGexConfig should have comment or be unexported